zuul.d/molecule.yaml

```diff
@@ -191,3 +201,14 @@
       - ^molecule-requirements.txt
       - ^ci_framework/roles/test_deps/(?!meta).*
       - ^ci/playbooks/molecule.*
+
```

Comment:
o_O why is it twice in there ?!

Reply:
Cause I duplicated it cause I didn't double-checked that the make target already added it :)